content/applications/websites/website/configuration/google_search_console.rst

```diff
@@ -7,21 +7,160 @@ monitor, maintain, and troubleshoot their site's presence in Google Search resul
+<https://search.google.com/search-console/welcome>`_, and click on :guilabel:`+ Add property`. If it
+is your first time doing it, you will see a window pop-up as soon as you open Google Search Console,
+asking you what property type you want to create.
+
+You have two options :
+
+- Domain property
+- URL prefix property
 
 .. image:: google_search_console/add-domain-or-url-prefix.png
    :alt: Google Search Console domain or URL prefix
 
-.. image:: google_search_console/verify.png
-   :alt: Verify domain or URL prefix
+Domain property
+===============
+
+.. note:: Google suggests creating at least one domain property to represent your site, as it is the
+   most comprehensive view of your website information. It will include all URLs in all subdomains
+   and across HTTP and HTTPS. It requires you to add the DNS record in your domain name provider,
+   but even if it takes some time, it is worth it.
+
+URL prefix property
+===================
+
+This type of verification is usually simpler as you have multiple verification methods, such as
+using your existing Google Analytics or Tag Manager account. It also makes sense to view a section
+of your website separately. For example, if you work with a consultant on a specific part of your
+website, you might want to verify this part separately to limit access to your data.
+
+.. note:: The best method for you depends on your comfort level and technical expertise. For
+   beginners, using a file upload or HTML tag might be easiest. Those options are convenient if you
+   already use Google Analytics or Tag Manager. You'll need to access your domain registrar's settings
+   for domain verification.
+
+
+Five ways to verify your site ownership
+=======================================
+
+.. _GSC-DNS-Record:
+
+DNS Record
+----------
+
+With this method, you add a DNS TXT record to your domain's DNS settings. Google provides you with a
+unique string to add as a TXT record. Once added, Google verifies ownership by checking the DNS
+settings.
+Start by copying the text record from the dialog google-site-verification=123aBc… after entering
+your domain.
+After you change your DNS record, go back to Search Console and click Verify in the verification pane.
+Once again, if the verification does not work immediately, wait a few hours or a day and try again,
+as the change may take some time to take effect. Just choose the unverified property from the
+:guilabel:`Properties` drop-down and it will try to verify you automatically.
+
+.. important:: To stay verified, do not remove the DNS Record even after verification succeeds.
+
+.. _GSC-HTML-File-Upload:
+
+HTML File Upload
+----------------
+
+This method involves uploading an HTML file provided by Google to the root directory of your website.
+Google verifies ownership by checking for this file. You will typically download the HTML verification
+file from Google Search Console, and then upload it to your website using FTP or file manager provided
```

Comment:
Hi @masp-odoo , Actually. for this part, could you remove it?
" using FTP or file manager provided
by your web hosting service. Odoo is the website builder that allows web hosting service on
:doc:`/administration/odoo_online` and :doc:`Odoo.sh </administration/odoo_sh/overview/introduction>`.
If you are using Odoo Online, it is a :abbr:`SaaS (Software as a service)` solution, meaning that it
is not :abbr:`FTP (File Transfert Protocol)`. Odoo Online provides private databases which are fully
managed and hosted by Odoo"

The parts before and after are fine. This one looks unnecessary because there is no option in Odoo called "FTP". In addition, the file upload through copy/paste of the google123abc.html code is actually type http.
https://github.com/odoo/odoo/blob/dcd426df41db9200b95d04ba664441053281db78/addons/website/controllers/main.py#L784
It's not relevant to describe FTP here for Odoo. Let's also not talk about Odoo Online and Odoo SH to maintain the topic about the HTML File Upload.

Thank you.